README.md

```diff
@@ -378,14 +378,14 @@ If you want to inspect emails, you can run fxa with [MailDev](https://www.npmjs.
 #### Install
 
 ```bash
-npm install maildev -g
+yarn maildev -g
```

Comment:
I don't think this is the correct equivalent in Yarn. I believe it would be `yarn global add maildev`

Reply:
Corrected. please let me in case any more updates are needed. :)